src/engine/create-changeset/tasks/index.ts

```diff
@@ -0,0 +1,13 @@
+import { affectedContentTypesDivergedTask } from './affected-content-types-diverged-task'
+import { computeIdsTask } from './compute-ids-task'
+import { fetchPartialEntitiesTask } from './fetch-partial-entities-task'
+import { fetchAddedEntitiesTask } from './fetch-added-entities-task'
+import { fetchChangedTasks } from './fetch-changed-tasks'
+
+export const createChangesetTasks = {
```

Comment:
I think I would name it `CreateChangeset`, because it"s the create Changeset namespace. This will give you following usage pattern `CreateChangeset. fetchChangedTasks({...})`. Wdyt?